addon/utils/messages.js

```diff
@@ -9,6 +9,10 @@ const assign = Ember.assign || Ember.merge;
 const Messages = assign({}, _Messages);
 
 export default assign(Messages, {
+  // Blank and present are flipped in ember-validators. Need to flip them back here
+  blank: '{description} must be blank',
+  present: '{description} can\'t be blank',
```

Comment:
I suggest doing the following:

```js
blank: _Messages.present,
present: _Messages.blank
```

That way if the error messages do change in the future, we wouldnt have to go back and change it here too.